vignettes/opart.Rmd

```diff
@@ -184,19 +235,78 @@ timing.dt$time <- timing.dt$time * (10^(-9))
 
 ```{r message = FALSE, fig.height = 5, fig.width = 7}
 
-p <- ggplot(data = timing.dt, aes(x = log10(size),y = log10(time), 
-                                  label=rownames(timing.dt),
+p <- ggplot(data = timing.dt, aes(x = (size),y = (time),
                                   col = expr))+
 
 geom_smooth(se=F)+
-#scale_x_log10() + scale_y_log10() +
-labs(x="size of data vector", y="log(time(s))", col="method")+
+
+labs(x="size of data vector on log scale", y="time(s) on log scale", col="method")+
   
-geom_hline(yintercept=log10(1), color="yellow", linetype = "dashed")+ 
-geom_hline(yintercept=log10(60), color="green", linetype = "dashed")+
-annotate(geom="text", x=2900, y=log10(60), label="1 minute")+
-annotate(geom="text", x=2900, y=log10(1), label="1 second")
+geom_hline(yintercept=(1), color="yellow", linetype = "dashed")+ 
+geom_hline(yintercept=(60), color="green", linetype = "dashed")+
+geom_hline(yintercept=(3600), color="orange", linetype = "dashed")+
+annotate(geom="label", x=(2900), y=(60), label="1 minute")+
```

Comment:
please use one geom_hline and one geom_text instead of three geom_hline and three annotate, e.g.
```
lab.df <- data.frame(
  seconds=c(1, 60, 60*60),
  label=c("1 second", "1 minute", "1 hour"))
ggplot()+
  geom_hline(aes(yintercept=log10(seconds)),
             data=lab.df,
             color="grey")+
  geom_text(aes(2.5, log10(seconds), label=label),
            data=lab.df,
            size=3,
            color="grey",
            vjust=-0.5)+
```
(taken from https://github.com/tdhock/PeakSegFPOP-paper/blob/master/figure-PDPA-timings.R#L77)